src/kakarot/library.cairo

```diff
@@ -252,14 +252,11 @@ namespace Kakarot {
         pedersen_ptr: HashBuiltin*,
         range_check_ptr,
         bitwise_ptr: BitwiseBuiltin*,
-    }(origin: felt, bytecode_len: felt, bytecode: felt*) -> (
-        starknet_contract_address: felt, evm_contract_address: felt
+    }(origin: felt, evm_contract_address: felt, bytecode_len: felt, bytecode: felt*) -> (
+        starknet_contract_address: felt
```

Comment:
add `value` here as args to pass it to `execute`

Reply:
done.